showcase/app/templates/utilities/dialog-primitive.hbs

```diff
@@ -0,0 +1,265 @@
+{{! @glint-nocheck: not typesafe yet }}
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "DialogPrimitive Components"}}
+
+<Shw::Text::H1>DialogPrimitive</Shw::Text::H1>
+
+<section data-test-percy>
+
+  <Shw::Text::H2>DialogPrimitive::Wrapper</Shw::Text::H2>
+
+  <Shw::Text::H3>With generic content</Shw::Text::H3>
+
+  <Shw::Flex {{style gap="2rem"}} as |SF|>
+    <SF.Item @label="Header + Body + Footer">
+      <div class="shw-utility-dialog-primitive-base-container">
+        <Hds::DialogPrimitive::Wrapper open>
+          <:header>
+            <Shw::Placeholder @text="header" @height="60px" />
+          </:header>
+          <:body>
+            <Shw::Placeholder @text="body" @height="300px" @background="#fafafa" tabindex="0" />
```

Comment:
should the `tabindex` be added to the `Hds::DialogPrimitive::Body` element instead? to make sure when there is longer content users can access the overflowing elements

Reply:
I thought about it, but I checked and currently nor the [`Flyout::Body`](https://github.com/hashicorp/design-system/blob/main/packages/components/src/components/hds/flyout/body.hbs#L6) nor the [`Modal::Body`](https://github.com/hashicorp/design-system/blob/main/packages/components/src/components/hds/modal/body.hbs#L6) have it. 

Done in https://github.com/hashicorp/design-system/pull/2211/commits/b65e14f170c01894485def4a3235204db8c90f4f

Reply:
Yeah, I think it's something the automated tests didn't pick up at the time, but I can see how it helps, so thanks for adding them!